docs/reference/discovery.mdx

```diff
@@ -62,8 +62,8 @@ Setup still requires configuration of the Discovery endpoint, but when using the
 import { config } from "@onflow/fcl"
 
 config({
-  "accessNode.api": "https://rest-testnet.onflow.org", // Mainnet: "https://rest-mainnet.onflow.org"
```

Comment:
No mainnet urls? Should we link our or have them in a diff section?

Reply:
@srinjoyc Is that relevant to the Discovery page?